cloudslang-content-verifier/src/main/java/io/cloudslang/lang/tools/build/verifier/SlangContentVerifier.java

```diff
@@ -47,37 +42,28 @@
     @Autowired
     private ScoreCompiler scoreCompiler;
 
-    @Autowired
-    private Yaml yaml;
-
     public Map<String, Executable> createModelsAndValidate(String directoryPath) {
         Validate.notEmpty(directoryPath, "You must specify a path");
         Validate.isTrue(new File(directoryPath).isDirectory(), "Directory path argument \'" + directoryPath + "\' does not lead to a directory");
         Map<String, Executable> slangModels = new HashMap<>();
-        Collection<File> slangFiles = FileUtils.listFiles(new File(directoryPath), SLANG_FILE_EXTENSIONS, true);
+        Collection<File> slangFiles = listFiles(new File(directoryPath), SLANG_FILE_EXTENSIONS, true, PROPERTIES_FILE_EXTENSIONS);
         log.info("Start compiling all slang files under: " + directoryPath);
         log.info(slangFiles.size() + " .sl files were found");
         log.info("");
         int ignoredExecutables = 0;
         for(File slangFile: slangFiles){
             Validate.isTrue(slangFile.isFile(), "file path \'" + slangFile.getAbsolutePath() + "\' must lead to a file");
-            if (isSystemPropertyFile(SlangSource.fromFile(slangFile))) {
-                ++ignoredExecutables;
-                log.info("Ignoring file: " + slangFile.getName() + " (not a valid executable)." +
-                        " Top level key '" + SlangTextualKeys.SYSTEM_PROPERTY_KEY + "' found instead of '" + SlangTextualKeys.FLOW_TYPE + "'/'" + SlangTextualKeys.OPERATION_TYPE + "'.");
-            } else {
-                Executable sourceModel;
-                try {
-                    sourceModel = slangCompiler.preCompile(SlangSource.fromFile(slangFile));
-                } catch (Exception e) {
-                    String errorMessage = "Failed creating Slang models for file: \'" + slangFile.getAbsoluteFile() + "\'.\n" + e.getMessage();
-                    log.error(errorMessage);
-                    throw new RuntimeException(errorMessage, e);
-                }
-                if (sourceModel != null) {
-                    staticSlangFileValidation(slangFile, sourceModel);
-                    slangModels.put(getUniqueName(sourceModel), sourceModel);
-                }
+            Executable sourceModel;
+            try {
+                sourceModel = slangCompiler.preCompile(SlangSource.fromFile(slangFile));
+            } catch (Exception e) {
+                String errorMessage = "Failed creating Slang models for file: \'" + slangFile.getAbsoluteFile() + "\'.\n" + e.getMessage();
+                log.error(errorMessage);
+                throw new RuntimeException(errorMessage, e);
+            }
+            if (sourceModel != null) {
+                staticSlangFileValidation(slangFile, sourceModel);
+                slangModels.put(getUniqueName(sourceModel), sourceModel);
```

Comment:
The duplication of code with the CLI keeps on growing...
I think we should deal with the common module refactor task already...